### ISSUE TITLE
Blank UI page after upgrade to 2.12.2 (ReverseProxy)

### ISSUE THREAD

**Issue by mischkasmith** [node-red/node-red-dashboard#460]
Hello.
I use node-red-dashboard in combination with apache reverse proxy.
Since the update from 2.11.0 to 2.12.2 the / ui page remains blank after loading. the /admin page  works normally.
After an downgrade to version 2.11.0 /ui works again.

This is my apache config.

```
RewriteEngine On
redirectmatch ^/red$ /red/

RewriteCond %{REQUEST_URI} ^/red/comms [NC] 
RewriteCond %{QUERY_STRING} transport=websocket [NC]
RewriteRule /(.*) ws://localhost:1880/$1 [P,L] 

ProxyRequests off
ProxyPreserveHost on
ProxyPass               "/red/comms"    "ws://localhost:1880/red/comms"
ProxyPassReverse        "/red/comms"    "ws://localhost:1880/red/comms"
ProxyPass               "/red/"         "http://localhost:1880/red/"
ProxyPassReverse        "/red/"         "http://localhost:1880/red/"
```

- [ ] Node-RED-Dashboard version:  2.12.2
- [ ] Node-RED version: 19.5
- [ ] node.js version: 10.13.0
- [ ] npm version: 6.4.1
- [ ] Platform/OS: raspian
- [ ] Browser: Chromium / Firefox / Chrome(android)

**Comment by dceejay** [comment]
I don't believe anything has changed in the comms area - are you sure you fully flushed the browser caches to reload the new dashboard contents ?

**Comment by dceejay** [comment]
I suspect you may be better asking for help on the wider Node-RED discourse form as I don't use Apache reverse proxy so won't be able to re-create this, or provide any advice where to look.

**Comment by dceejay** [comment]
closed by 5f6b111b02e51b342dd8281acf458cba9f18c24b
